src/main/groovy/com/twcable/grabbit/server/batch/steps/jcrnodes/JcrNodesProcessor.groovy

```diff
@@ -59,152 +49,31 @@ class JcrNodesProcessor implements ItemProcessor<JcrNode, ProtoNode> {
     @Nullable
     ProtoNode process(JcrNode jcrNode) throws Exception {
 
+        JCRNodeDecorator decoratedNode = new JCRNodeDecorator(jcrNode);
+
         //TODO: Access Control Lists nodes are not supported right now.
-        if (!jcrNode || jcrNode.path.contains("rep:policy")) {
-            log.info "Ignoring current node ${jcrNode}"
+        if (!decoratedNode || decoratedNode.path.contains("rep:policy")) {
```

Comment:
I'm not sure Spring will ever send us a null reference to jcrNode, and if it did we would have issues with the JCRNodeDecorator constructor above so we wouldn't get here anyway.